flink-runtime/src/test/java/org/apache/flink/runtime/state/IncrementalKeyedStateHandleTest.java

```diff
@@ -187,7 +188,7 @@ public void testSharedStateDeRegistration() throws Exception {
 
 	private static IncrementalKeyedStateHandle create(Random rnd) {
 		return new IncrementalKeyedStateHandle(
-			"test",
+			UUID.nameUUIDFromBytes("test".getBytes()),
```

Comment:
We need to always pass an explicit charset when converting between strings/bytes:
```java
"test".getBytes(StandardCharsets.UTF_8);
```